is-valid/src/main.rs

```diff
@@ -0,0 +1,19 @@
+use failure::Error;
+use std::io::stdin;
+
+use hpos_config_core::{config::Admin};
+fn is_valid() -> Result<(), Error> {
+
+    let admin: Admin = serde_json::from_reader(stdin())?;
+    Ok(())
+
+}
+fn main() -> Result<(), Error> {
+    std::process::exit(match is_valid() {
+        Ok(_) => 0,
+        Err(err) => {
+            eprintln!("error: {:?}", err);
+            1
+        }
+    });
+}
```

Comment:
This is an incorrect approach. Literally all you want is:

```rust
fn main() -> Fallible<()> {
    let Config::V1 { .. } = serde_json::from_reader(stdin())?
}
```

Reply:
Done thank you for that